Dockerfile

```diff
@@ -6,4 +6,4 @@ COPY requirements.txt /requirements.txt
 RUN pip install --upgrade pip
 RUN pip install -r requirements.txt
 
-CMD uvicorn app.simple:app --host 0.0.0.0
+CMD uvicorn app.chessapi:app --host 0.0.0.0
```

Comment:
are you completely sure about the "app.chessapi:app" part? I am a bit confused because I think one of them reflects the folder that the app is in and the other refers to the name of the app within the chessapi.py. But we don't have an app folder, do we?